plugins/amoeba/tests/TestAmoebaGeneralizedKirkwoodForce.h

```diff
@@ -96,63 +97,57 @@ static void setupMultipoleAmmonia(System& system, AmoebaGeneralizedKirkwoodForce
     std::vector<double> nitrogenMolecularDipole(3);
     std::vector<double> nitrogenMolecularQuadrupole(9);
 
-    nitrogenMolecularDipole[0]     =   8.3832254e-03;
-    nitrogenMolecularDipole[1]     =   0.0000000e+00;
-    nitrogenMolecularDipole[2]     =   3.4232474e-03;
-
-    nitrogenMolecularQuadrupole[0] =  -4.0406249e-04;
-    nitrogenMolecularQuadrupole[1] =   0.0000000e+00;
-    nitrogenMolecularQuadrupole[2] =  -2.6883671e-04;
-    nitrogenMolecularQuadrupole[3] =   0.0000000e+00;
-    nitrogenMolecularQuadrupole[4] =   2.5463927e-04;
-    nitrogenMolecularQuadrupole[5] =   0.0000000e+00;
-    nitrogenMolecularQuadrupole[6] =  -2.6883671e-04;
-    nitrogenMolecularQuadrupole[7] =   0.0000000e+00;
-    nitrogenMolecularQuadrupole[8] =   1.4942322e-04;
+    nitrogenMolecularDipole[0]     =  0.0;
+    nitrogenMolecularDipole[1]     =  0.0;
+    nitrogenMolecularDipole[2]     =  0.01555516;
```

Comment:
I would feel better about adding a new test rather than replacing the existing one.  We need to make sure the existing code continues to produce the same results it did before.